src/apps/common/submodules/dragableUploads/dragableUploads.js

```diff
@@ -181,6 +184,8 @@ define(function(require) {
 		dragableUploadsRequestMediaUpload: function(callback, file) {
 			var self = this;
 
+			console.log('file', file);
```

Comment:
You forgot to remove the console.log line

Reply:
ouch, thanks!